includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -432,10 +432,11 @@ public function stripe_check_environment() {
 
 				$oauth_required = $needs_live_oauth || $needs_test_oauth;
 				if ( $oauth_required ) {
-					$message = __( 'Please reconnect to continue using Stripe and avoid disruptions on your store.', 'woocommerce-gateway-stripe' );
-					$link    = esc_url( admin_url( 'admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings&highlight=account-details' ) );
-					$actions = [
-						'<a href="' . $link . '" style="display: block; padding: 10px; color: #007cba; border: #007cba solid 1px; float: left; text-decoration: none; margin-bottom: 12px; border-radius: 2px;">'
+					$message  = __( 'Please reconnect to continue using Stripe and avoid disruptions on your store.', 'woocommerce-gateway-stripe' );
+					$link     = esc_url( admin_url( 'admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings&highlight=account-details' ) );
```

Comment:
The code in `client/settings/payment-settings/account-details-section.js` is expecting a `track` url param:

```suggestion
					$link     = esc_url( admin_url( 'admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings&highlight=account-details&track=true' ) );
```